build_tools/lightweight_dispatch_ops.yaml

```diff
@@ -0,0 +1,741 @@
+- aten::to.dtype_layout
+- aten::to.other
+- aten::to.prim_Device
+- aten::to.device
+- aten::type_as
+- aten::view
+- aten::_unsafe_view
+- aten::where.self
+- aten::where.Scalar
+- aten::where.ScalarOther
+- aten::where.ScalarSelf
+- aten::slice.Tensor
+- aten::len.Tensor
+- aten::cpu
+- aten::gather
+- aten::scatter_add
+- aten::IntImplicit
+- aten::tensor.float
+- aten::Int.Tensor
+- aten::Float.Tensor
+- aten::dropout
+- aten::native_dropout
+- aten::t
+- aten::numpy_T
+- aten::full
+- aten::full_like
+- aten::baddbmm
+- aten::alias_copy
+- aten::as_strided_copy
+- aten::diagonal_copy
+- aten::expand_copy
+- aten::permute_copy
+- aten::_reshape_alias_copy
+- aten::select_copy.int
+- aten::detach_copy
+- aten::slice_copy.Tensor
+- aten::squeeze_copy
+- aten::squeeze_copy.dim
+- aten::t_copy
+- aten::transpose_copy.int
+- aten::unsqueeze_copy
+- aten::view_copy
+- aten::view_copy.dtype
+- aten::unfold_copy
+- aten::select_scatter
+- aten::slice_scatter
+- aten::diagonal_scatter
+- aten::as_strided_scatter
+- aten::__contains__.str
+- aten::__contains__.int_list
+- aten::__getitem__.Dict_str
+- aten::_set_item.str
+- aten::keys.str
+- aten::get_str
+- aten::Delete.Dict_str
+- aten::cat
+- aten::append.t
+- aten::add.t
+- aten::eq.int_list
+- aten::list.t
+- aten::slice.t
+- aten::insert.t
+- aten::ne.int_list
+- aten::any.bool
+- aten::add.str
+- aten::eq.str
+- aten::len.str
+- aten::str
+- aten::format
+- aten::join
+- aten::Float.Scalar
+- aten::Float.str
+- aten::Int.float
+- aten::Int.Scalar
+- aten::__range_length
+- aten::__derive_index
+- aten::gt.int
+- aten::ge.int
+- aten::lt.int
+- aten::le.int
+- aten::ne.int
+- aten::eq.int
+- aten::floordiv.int
+- aten::remainder.int
+- aten::remainder.Scalar
+- aten::add.int
+- aten::sub.int
+- aten::mul.int
+- aten::neg.int
+- aten::log.int
+- aten::add.float_int
+- aten::sub.float
+- aten::mul.float
+- aten::div.float
+- aten::neg.float
+- aten::eq.float
+- aten::gt.float
+- aten::ge.float
+- aten::lt.float
+- aten::lt.float_int
+- aten::ge.float_int
+- aten::ne.float_int
+- aten::gt.float_int
+- aten::__and__.bool
+- aten::ne.bool
+- aten::__is__
+- aten::__isnot__
+- aten::__not__
+- aten::len.t
+- aten::__getitem__.t
+- aten::_set_item.t
+- aten::div
+- aten::add
+- aten::sqrt.int
+- aten::Bool.float
+- aten::Bool.int
+- aten::eq.device
+- aten::ceil.float
+- aten::narrow
+- aten::ScalarImplicit
+- aten::_softmax_backward_data
+- aten::tanh_backward
+- aten::gelu_backward
+- aten::_log_softmax_backward_data
+- aten::native_layer_norm_backward
+- aten::embedding_dense_backward
+- aten::native_batch_norm_backward
+- aten::native_dropout_backward
+- prim::layout
+- prim::TupleIndex
+- prim::device
+- prim::dtype
+- prim::TupleUnpack
+- prim::NumToTensor.Scalar
+- prim::min.self_int
+- prim::min.int
+- prim::max.self_int
+- prim::max.int
+- prim::RaiseException
+- prim::Uninitialized
+- prim::unchecked_cast
+- prim::Print
+- prim::tolist
+- prim::abs.Scalar
+# torch-mlir ops end
```

Comment:
Below here are ops that are imported somewhere in torch python sources files (and thus need to be linked/registered/etc).